### ISSUE TITLE
Example in readme doesn't work as is

### ISSUE THREAD

**Issue by derek-adair** [aio-libs/aiohttp#556]
Spent a healthy chunk of time trying to debug the example in the readme... doesn't seem to work as is.  Maybe i'm missing something simple in how I am configuring my environment? or perhaps this isn't meant to be an actual functional example?

```
(py3.4)➜  asyncio git:(build-dev) ✗ python app.py
  File "app.py", line 5
    async def handle(request):
        ^
```

The basic_srv example worked fine for me copy/pasta style, but i get a syntax error on that demo... is there something obvious to add to that demo to make it work?

**Comment by asvetlov** [comment]
It uses syntax available starting from python 3.5
For python 3.4 you should use `@coroutine/yield from` instead of `async def/await`.

**Comment by asvetlov** [comment]
Fixed by 6f4a0ff
